latest/index.bs

```diff
@@ -288,14 +288,50 @@ It MAY contain exactly one `translation` that specifies the offset from the orig
 The length of the `scale` and `translation` array MUST be the same as the length of "axes".
 The requirements (only `scale` and `translation`, restrictions on order) are in place to provide a simple mapping from data coordinates to physical coordinates while being compatible with the general transformation spec.
 
-Each "multiscales" dictionary MAY contain the field "coordinateTransformations", describing transformations that are applied to all resolution levels in the same manner.
-The transformations MUST follow the same rules about allowed types, order, etc. as in "datasets:coordinateTransformations" and are applied after them.
-They can for example be used to specify the `scale` for a dimension that is the same for all resolutions.
-
-Each "multiscales" dictionary SHOULD contain the field "name". It SHOULD contain the field "version", which indicates the version of the multiscale metadata of this image (current version is [NGFFVERSION]).
-
-Each "multiscales" dictionary SHOULD contain the field "type", which gives the type of downscaling method used to generate the multiscale image pyramid.
-It SHOULD contain the field "metadata", which contains a dictionary with additional information about the downscaling method.
+Each "multiscales" dictionary:
+
+<ul>
+<li>
+   MAY contain the field "coordinateTransformations", describing transformations that are applied to all resolution levels in the same manner.
+   The transformations MUST follow the same rules about allowed types, order, etc. as in "datasets:coordinateTransformations" and are applied after them.
+   They can for example be used to specify the `scale` for a dimension that is the same for all resolutions.
+</li>
+
+<li>
+   SHOULD contain the field "name" which is informative to users but not sufficient for unique identifiation.
+</li>
+
+<li>
+  <p>
+   SHOULD contain the field "uuid" which is a
```

Comment:
Could we also / alternatively use a [content identifier (CID)](https://py-cid.readthedocs.io/en/master/)? This can help prevent duplicates (it can be generated from the data).

Reply:
Nevermind, after further thought, modifying the content will change the CID.

But, it worth noting that with content-addressed storage, there is no need for a uuid.

Reply:
Guess the question to you is how long until you think content-addressed storage is main stream :)

More seriously, do you think this should be made flexible/configurable so that a CID could be stored in place of a UUID?